### ISSUE TITLE
docs are broken

### ISSUE THREAD

**Issue by jgangemi** [fabric8io/docker-maven-plugin#468]
the `run` section is missing from the `docker:start` page.

**Comment by rhuss** [comment]
Hmm. Not sure what you mean :) That section -> https://fabric8io.github.io/docker-maven-plugin/docker-start.html#-run-configuration ?

**Comment by jgangemi** [comment]
no, it's more this section that i was looking for: https://fabric8io.github.io/docker-maven-plugin/docker-build.html#start-up-arguments

now i realize it's on the `docker:build` page. while i do see the link on the page now, maybe we could add a small section, similar to `port mappings`, etc to the `docker:start` page that specifically calls this out.

**Comment by rhuss** [comment]
yeah, I see this now. will fix it asap.
